pkg/controller/v1beta1/inferenceservice/reconcilers/deployment/deployment_reconciler.go

```diff
@@ -99,15 +109,18 @@ func (r *DeploymentReconciler) checkDeploymentExist(client kclient.Client) (cons
 		return constants.CheckResultUnknown, nil, err
 	}
 	// existed, check equivalence
-	// for HPA scaling, we should ignore Replicas of Deployment
-	ignoreFields := cmpopts.IgnoreFields(appsv1.DeploymentSpec{}, "Replicas")
+	cmpOpts := cmp.Options{}
```

Comment:
isn't better to keep `ignoredFields` bot a better readability?

Reply:
I think I used `cmdOpts` as it may be empty. But I'll change back to ignoreFields for now. I  have no problem with that naming.